qa/L0_perf_analyzer/test.sh

```diff
@@ -63,8 +63,6 @@ for CONFIG_FILE in ${LIST_OF_CONFIG_FILES[@]}; do
         TEST_NAME="count_window"
     elif [[ "$CONFIG_FILE" == "config-additive-args-count-no-adjust.yml" ]]; then
         TEST_NAME="count_window"
-    elif [[ "$CONFIG_FILE" == "config-perf-output-timeout.yml" ]]; then
```

Comment:
Is this no longer valid?

Reply:
Correct. This checked that the summary table was empty. Now it will fail because we have two consecutive PA timeouts. I've made this a unit test now.